ExpressJS/utils/sequilize.js

```diff
@@ -0,0 +1,172 @@
+require('dotenv').config({ path: './utils/dev.env' });
```

Comment:
Do we actually need to pass the path again here if it will be the same .env of the root folder?